state_machine/applications/flight/Tasks/blink_task.py

```diff
@@ -11,8 +12,8 @@ class task(Task):
 
     async def main_task(self):
         if self.rgb_on:
-            self.cubesat.RGB = (0, 0, 0)
+            cubesat.set_RGB((0, 0, 0))
```

Comment:
set_RGB(0, 0, 0) -> RGB(0, 0, 0)

Update on this function: I refactored the new driver so we have only one RGB function. If we pass no value to this function, it'll return the current RGB value. If we pass a value, it'll set RGB to the passed value.